.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "**/*": "npm run format"
```

Comment:
is this glob "**/*" efficient? Just wondering how long it takes to scan all these files?

Reply:
Should it be like this?

```
{
  "*.ts": ["prettier --write", "eslint"],
  "*.tsx": ["prettier --write", "eslint"],
  "*.html": ["eslint", "prettier --write"],
  "*.scss": "prettier --write"
}

```

Reply:
`format` will run both `prettier:fix` and `lint:fix`, see [this](https://github.com/nodejs/nodejs.org/blob/main/package.json#L26)

Reply:
I know what `format` does, just wondering if `**/*` is an efficient glob. We don't want our pre-commit hook to take forever. But I think it's fine.

Reply:
Ah, actually I was replying to @ashutosh887 👀